shoop_tests/admin/test_order_creator.py

```diff
@@ -259,15 +259,21 @@ def test_company_contact_creation(rf, admin_user):
 
 
 def test_editing_existing_order(rf, admin_user):
+    modifier = UserFactory()
     get_initial_order_status()  # Needed for the API
     contact = create_random_person(locale="en_US", minimum_name_comp_len=5)
     state = get_frontend_order_state(contact=contact)
     shop = get_default_shop()
     order = create_empty_order(shop)
+    order.payment_data = {"payment_data": True}
+    order.shipping_data = {"shipping_data": True}
+    order.extra_data = {"external_id": "123"}
     order.save()
     assert order.lines.count() == 0
     assert order.pk is not None
-    request = get_frontend_request_for_command(state, "finalize", admin_user)
+    assert order.modified_by is None  # Order is not edited
+    assert order.modified_on is None  # Order is not edited
```

Comment:
Shouldn't modified by be initialized to the same value as creator? I.e. these asserts should be
```python
assert order.modified_by == order.creator
assert order.modified_on == order.order_date
```